package.json

```diff
@@ -111,6 +113,16 @@
     "extends": [
       "react-app",
       "react-app/jest"
+    ],
```

Comment:
does this work if you add it to `.eslintrc` instead? we should keep all rules there if possible

Reply:
Let me try, I guess this was added automatically